src/qibo/transpiler/placer.py

```diff
@@ -319,6 +323,7 @@ def __call__(self, circuit):
         Returns:
             (dict): physical-to-logical qubit mapping.
         """
+        random.seed(self.seed)
```

Comment:
Instead of using the `random` package for this, I'd recommend allowing `numpy.random.Generator`s. You can take a look on how its done in other modules here in `qibo` (e.g. `quantum_info.random_ensembles`).